linux_os/guide/system/software/updating/clean_components_post_updating/rule.yml

```diff
@@ -1,13 +1,17 @@
 documentation_complete: true
 
-prodtype: ol7,ol8,rhel7,rhel8,rhv4
```

Comment:
This file has been updated in another pull request: https://github.com/ComplianceAsCode/content/pull/6631/files#diff-f0247c1c3e940c8b4c066c11094c9875a15cd3d0f0563929cf3a79f360c38349

Reply:
I think you can leave it here and I'll comment in the #6631 to remove it there

Reply:
Let's remove this one and keep the one from #6631 since it contains newer changes